### PR TITLE
16476: Edit/Delete IM1 integrations E2E tests

### PR DESCRIPTION
**Pull request by PhysicsJon**

### CODE REVIEW

File: tests/NHSD.GPIT.BuyingCatalogue.E2E.PublicBrowseTests/Areas/Admin/AddNewSolution/Interoperability/EditIm1Integration.cs [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+            var solution = await context.Solutions.SingleAsync(s => s.CatalogueItemId == SolutionId);
+            var integration = solution.GetIntegrations().Single(i => i.Id == IntegrationId);
+            integration.Qualifier.Should().BeEquivalentTo(integrationType);
+            integration.IsConsumer.Should().Be(providerConsumer == "Consumer");
+            integration.IntegratesWith.Should().BeEquivalentTo(integratesWith);
+            integration.Description.Should().BeEquivalentTo(description);
+        }
+
+        [Fact]
+        public void EditIm1Integration_MissingIntegratesWithValue()
```
Comment (leerogers-bjss) [inline tests/NHSD.GPIT.BuyingCatalogue.E2E.PublicBrowseTests/Areas/Admin/AddNewSolution/Interoperability/EditIm1Integration.cs:84]: you need to check that both select lists and both input fields generate the correct error. not just that an error is generated :)

**Review by leerogers-bjss — CHANGES_REQUESTED** [review]

File: tests/NHSD.GPIT.BuyingCatalogue.E2E.PublicBrowseTests/Areas/Admin/AddNewSolution/Interoperability/EditIm1Integration.cs (hunk 1 above)
Comment (PhysicsJon) [reply tests/NHSD.GPIT.BuyingCatalogue.E2E.PublicBrowseTests/Areas/Admin/AddNewSolution/Interoperability/EditIm1Integration.cs:84]: This is the same page as Adding IM1 integrations, those tests cover that functionality and I didn't want to duplicate too many tests, hence a single smoke test here

**Review by leerogers-bjss — APPROVED** [review]
